src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -1210,8 +1213,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Public Shared Function ClassifyTupleConversion(source As BoundTupleLiteral, destination As TypeSymbol, binder As Binder, <[In], Out> ByRef useSiteDiagnostics As HashSet(Of DiagnosticInfo)) As ConversionKind
+            If source.Type = destination Then
+                Return ConversionKind.Identity
+            End If
+
             Dim arguments = source.Arguments
 
+            Dim wideningConversion = ConversionKind.WideningTuple
+            Dim narrowingConversion = ConversionKind.NarrowingTuple
+
+            If destination.IsNullableType Then
+                destination = destination.GetNullableUnderlyingType()
+
+                wideningConversion = ConversionKind.WideningNullable
+                narrowingConversion = ConversionKind.NarrowingNullable
+            End If
+
+            ' tuple literal converts to its inferred type 
+            If source.InferredType?.IsSameTypeIgnoringCustomModifiers(destination) Then
+                Return wideningConversion
```

Comment:
Identity would be handled at the entrance to the method. 

Here we have a case where the target type is different from the type natural type - it could be because target is nullable or tuple has no type at all while having a natural type. 

We know we have a trivial conversion, but we also know that it is not to the same type - so it is Widening.